yarn-audit-known-issues

```diff
@@ -1,5 +1,8 @@
-https://npmjs.com/advisories/1316
-https://npmjs.com/advisories/1324 
-https://npmjs.com/advisories/1084
-https://npmjs.com/advisories/1325
-https://npmjs.com/advisories/1179
+https://npmjs.com/advisories/1316
+https://npmjs.com/advisories/1324
+https://npmjs.com/advisories/1084
+https://npmjs.com/advisories/1325
+https://npmjs.com/advisories/1179
+
+https://npmjs.com/advisories/1523
```

Comment:
1523 (low severity) - No fix is currently available

Reply:
bit weird that the diff is showing up all the other lines =/